content/tutorials/esc-github/_index.md

```diff
@@ -0,0 +1,189 @@
+
+```yaml
+values:
+  aws:
+    login:
+      fn::open::aws-login:
+        oidc:
+          duration: 1h
+          roleArn: arn:aws:iam::123456789012:role/aws-esc-role
+          sessionName: esc-${context.pulumi.user.login}
+  environmentVariables:
+    AWS_ACCESS_KEY_ID: ${aws.login.accessKeyId}
+    AWS_SECRET_ACCESS_KEY: ${aws.login.secretAccessKey}
+    AWS_SESSION_TOKEN: ${aws.login.sessionToken}
+```
+
+{{< figure src="esc-add-env.png" caption="Figure: Pulumi ESC add environment">}}
+
+{{% notes type="warning" %}}
+Replace the roleArn value with the ARN of your AWS IAM role configured for Pulumi Cloud OIDC access. See the [configuring OIDC guide](/docs/pulumi-cloud/access-management/oidc/provider/aws/ for details.
+{{% /notes %}}
+
+## Step 4: Add a GitHub Actions workflow to deploy your stack
+
+Now you'll create a GitHub Actions workflow that authenticates with Pulumi Cloud, retrieves AWS credentials from your ESC environment, and runs `pulumi up` to deploy your infrastructure.
+
+1. In your Pulumi project, create a new directory called `.github/workflows`
+
+```bash
+mkdir -p .github/workflows
+```
+
+## Step 4: Configure a deployment GitHub action
+
+1. In your Pulumi project, create a new directory called `.github/workflows`.
+2. Inside that directory, create a new file named `pulumi.yaml`.
+3. Add the following workflow configuration:
+
+```yaml
+name: Pulumi Deployment
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Authenticate with Pulumi Cloud
+        uses: pulumi/auth-actions@v1
+        with:
+          organization: tutorials
+          requested-token-type: urn:pulumi:token-type:access_token:organization
+
+      - name: Install and inject AWS credentials
+        uses: pulumi/esc-action@v1
+        with:
+          environment: '<your-organization>/<your-esc-env>'
+
+      - name: Verify environment variables
+        run: |
+          echo "AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID"
+
+      - name: Pulumi Preview
+        uses: pulumi/actions@v6
+        with:
+          command: preview
+          stack-name: <your-org>/<stack-project>/<stack>
+
+      - name: Pulumi Up
+        uses: pulumi/actions@v6
+        with:
+          command: up
+          stack-name: tutorials/pulumi-esc-github-action/dev
+          up-args: --yes
+```
+
+{{% notes type="info" %}}
+Replace <your-org>/<your-esc-env> with the name of your Pulumi ESC environment, and update stack-name values with your organization, project, and stack replacing stack-name: <your-org>/<stack-project>/<stack>.
```

Comment:
For some reason the <your-org>/<your-esc-env> just showing up as a `/`. Also <your-org>/<stack-project>/<stack> is `//`.  Maybe not escaped properly?